client/src/lib/reminders/hooks/useUpdatePracticeReminders.ts

```diff
@@ -18,19 +18,40 @@ import {
   DEFAULT_NUMBER_OF_PRACTICE_REMINDERS,
   REMINDER_INTERVALS,
 } from '../constants';
+import useUser from '../../user/hooks/useUser';
 
 dayjs.extend(utc);
 
 const ID_PREFIX = 'practice';
 
 const useUpdatePracticeReminders = () => {
   const {t} = useTranslation('Notification.PracticeReminder');
+  const user = useUser();
   const {pinnedCollections} = usePinnedCollections();
   const {completedCollectionEvents} = useUserEvents();
   const getCollectionById = useGetCollectionById();
   const {removeTriggerNotifications, setTriggerNotification} =
     useTriggerNotifications();
 
+  const resolveBody = useCallback(
+    (collection: Collection | null, index: number) => {
+      if (!user || user?.isAnonymous) {
```

Comment:
Anonymous users can have display names too. So it should check for that instead.

Reply:
Ah, right